packages/@react-spectrum/list/test/ListViewDnd.test.js

```diff
@@ -228,6 +228,74 @@ describe('ListView', function () {
         });
       });
 
+      it('should allow drag and drop of a single row, selectionMode="none"', async function () {
```

Comment:
I noticed this fix fixed `touch` events/devices too. I assume `usePress` normalizes the two event types enough that we don't need an explicate `pointerType` `touch` test?

Reply:
Yeah, we just have ListView DnD tests for mouse and keyboard now. The usePress tests handle touch, so I think we're good with that.